packages/grafana-ui/src/components/Modal/Modal.tsx

```diff
@@ -79,7 +80,9 @@ export function Modal(props: PropsWithChildren<Props>) {
             <IconButton surface="header" name="times" size="xl" onClick={onDismiss} />
           </div>
         </div>
-        <div className={cx(styles.modalContent, contentClassName)}>{children}</div>
+        <div className={cx(styles.modalContent, contentClassName)}>
+          <CustomScrollbar>{children}</CustomScrollbar>
```

Comment:
Does this add `overflow-y: hidden` and if so will it reintroduce the `<Select/>` inside `<Modal />` bug that was fixed recently?

Reply:
ah sorry this was me trying to experiment with fixing https://github.com/grafana/grafana/issues/36317 (in a generic way) before starting this fix, will remove